HPXMLtoOpenStudio/resources/misc_loads.rb

```diff
@@ -27,7 +27,15 @@ def self.apply_plug(model, plug_load, obj_name, living_space)
     if plug_load.location == HPXML::LocationExterior
       # Set all heat gain as lost
       sens_frac = 0.0
-      lat_fract = 0.0
+      lat_frac = 0.0 # FIXME: is lat_fract intended?
```

Comment:
Good catch, looks like a typo to me. Feel free to remove the FIXMEs here and below.